bindings/java/src/main/com/apple/foundationdb/Database.java

```diff
@@ -161,24 +161,147 @@ default Transaction createTransaction() {
 	 */
 	double getMainThreadBusyness();
 
+	/**
+	 * Runs {@link #purgeBlobGranules(Function)} on the default executor.
```

Comment:
nit: does this throw an error if the range to purge is not aligned to an existing range granule or active blob range?  If so, could you document that here?